docs/how-ubuntu-is-made/concepts/version-strings.md

```diff
@@ -335,27 +341,29 @@ Native in Debian:
 
 Native in Ubuntu:
 
-| Previous version    | Devel upload                 | SRU upload     |
-| --------------------| ---------------------------- | -------------- |
-| `2.0ubuntu0`        | `2.1ubuntu0` or `3.0ubuntu0` | `2.0ubuntu0.1` |
-| `2ubuntu0`          | `3ubuntu0`                   | `2ubuntu0.1`   |
+| Previous version               | Devel upload                   | SRU upload                              |
+|--------------------------------|--------------------------------|-----------------------------------------|
+| `2.0ubuntu`                    | `2.1ubuntu` or `3.0ubuntu`     | `2.0ubuntu0.1`                          |
```

Comment:
again - keep the old format on the left side and tell them what the new one should be then